h2/src/main/org/h2/expression/SequenceValue.java

```diff
@@ -80,26 +80,26 @@ public void updateAggregate(Session session, int stage) {
     @Override
     public boolean isEverything(ExpressionVisitor visitor) {
         switch (visitor.getType()) {
-        case ExpressionVisitor.EVALUATABLE:
-        case ExpressionVisitor.OPTIMIZABLE_AGGREGATE:
-        case ExpressionVisitor.NOT_FROM_RESOLVER:
-        case ExpressionVisitor.GET_COLUMNS1:
-        case ExpressionVisitor.GET_COLUMNS2:
-            return true;
-        case ExpressionVisitor.DETERMINISTIC:
-        case ExpressionVisitor.INDEPENDENT:
-        case ExpressionVisitor.QUERY_COMPARABLE:
-            return false;
-        case ExpressionVisitor.SET_MAX_DATA_MODIFICATION_ID:
-            visitor.addDataModificationId(sequence.getModificationId());
-            return true;
-        case ExpressionVisitor.GET_DEPENDENCIES:
-            visitor.addDependency(sequence);
-            return true;
-        case ExpressionVisitor.READONLY:
-            return current;
-        default:
-            throw DbException.throwInternalError("type="+visitor.getType());
+          case ExpressionVisitor.EVALUATABLE:
+          case ExpressionVisitor.OPTIMIZABLE_AGGREGATE:
+          case ExpressionVisitor.NOT_FROM_RESOLVER:
+          case ExpressionVisitor.GET_COLUMNS1:
+          case ExpressionVisitor.GET_COLUMNS2:
+              return true;
+          case ExpressionVisitor.DETERMINISTIC:
+          case ExpressionVisitor.INDEPENDENT:
+          case ExpressionVisitor.QUERY_COMPARABLE:
+              return false;
+          case ExpressionVisitor.SET_MAX_DATA_MODIFICATION_ID:
+              visitor.addDataModificationId(sequence.getModificationId());
+              return true;
+          case ExpressionVisitor.GET_DEPENDENCIES:
+              visitor.addDependency(sequence);
+              return true;
+          case ExpressionVisitor.READONLY:
+              return current;
+          default:
+              throw DbException.throwInternalError("type="+visitor.getType());
```

Comment:
Please, revert changes in this file. You corrupted the indentation somehow.

Reply:
Sorry, the merging using only github-browser-page .... Now I just copied your version.

Reply:
Right now I see extra 4 spaces inside the whole `switch` statement.